### ISSUE TITLE
Prompt engineering

### ISSUE THREAD

**Issue by aantn** [robusta-dev/kubernetes-chatgpt-bot#2]
Copying over this idea from the reddit post :)

If anyone is curious, you can change the prompt on [line 77](https://github.com/robusta-dev/kubernetes-chatgpt-bot/blob/main/chatgpt_robusta_actions/chat_gpt.py#L77) to tweak the answers.

For example, maybe you'll get better answers by including pod logs:

```
search_term=f"How to solve {alert_name} on Kubernetes when the log is {alert.pod.get_logs()}?"
```

Or maybe you'd prefer an answer from Captain Jack Sparrow:

```
search_term=f"Explain like a pirate how to solve {alert_name} on Kubernetes?"
```

PRs welcome!

**Comment by anushkabishnoi** [comment]
Hello, this is my first experience with open source contributions. Would really appreciate it if you explained exactly what is needed to be done.

I would love to work on this issue.

Thanks in advance! :)

**Comment by aantn** [comment]
I'd suggest getting started with the [tutorial on writing Robusta actions](https://docs.robusta.dev/master/tutorials/my-first-custom-action.html). Then start to modify the prompt in the place mentioned above. See if you can get the bot to give different responses. 

After that, ping me and we'll discuss next steps.

**Comment by dotsvm** [comment]
Hey @aantn did we need to change line 77
from this:
search_term=f"How to solve {alert_name} on Kubernetes when the log is {alert.pod.get_logs()}?"
to this:
search_term=f"Explain like a pirate how to solve {alert_name} on Kubernetes?"

please reply.

**Comment by aantn** [comment]
Yep, something like that! But it should be an option, based on the params object.

**Comment by Islam-Programmer** [comment]
Hello aantn!, i dont even know what is this library, so please i want to learn what is this and do i need to learn it.
if its useful i want to learn, can you tell me how and from where can i learn that
